tests/test_syntax/_test_seq_pe1.py

```diff
@@ -0,0 +1,405 @@
+import magma as m
```

Comment:
Could we move this to peak and get rid of the fault dependency? If there's not a specific reason to leave this in magma, I think it would just mechanically (CI, environment) be easier to not have the magma -> fault dependency.

Reply:
Yes, that's why it's named `_test_seq_pe1`. The `_` marks it to be skipped by pytest. The full test is now part of lassen. I was keeping this file around as a reference, but we can remove it.